src/main/kotlin/no/nav/syfo/personstatus/domain/PersonOversiktStatus.kt

```diff
@@ -13,14 +13,14 @@ import java.time.LocalDate
 import java.time.OffsetDateTime
 
 data class PersonOversiktStatus(
-    val veilederIdent: String?,
+    val veilederIdent: String? = null,
     val fnr: String,
-    val navn: String?,
-    val enhet: String?,
+    val navn: String? = null,
+    val enhet: String? = null,
     val motebehovUbehandlet: Boolean? = null,
     val oppfolgingsplanLPSBistandUbehandlet: Boolean? = null,
     val dialogmotesvarUbehandlet: Boolean = false,
-    val dialogmotekandidat: Boolean? = false,
+    val dialogmotekandidat: Boolean? = null,
```

Comment:
Har denne endringen noe å si? Eller bryr ikke koden seg om det er null eller false senere?

Reply:
Av det jeg kunne se så sendte vi inn `null` der vi instantierte opp `PersonOversiktStatus`. (Se for eks. endringen i `Aktivitetskrav`